src/services/rfqm_service.ts

```diff
@@ -321,7 +324,7 @@ export class RfqmService {
         const opts: RfqmRequestOptions = {
             ...RFQM_DEFAULT_OPTS,
             txOrigin: this._registryAddress,
-            apiKey,
+            apiKey: integratorId, // Send the integrator ID vice the api key to the market makers
```

Comment:
nit: what does `vice` mean in this context? Possible this should be `via` instead? Also, would be nice to leave a TODO here just like we have on the `RFQT` side

Reply:
ugh lol https://www.grammarphobia.com/blog/2009/09/vice-isnt-nice.html

Reply:
we shouldn't need a TODO because this won't go away